docs/user/recom.rst

```diff
@@ -267,6 +267,67 @@ while also being sensitive to the municipalities
    </div>
 
 
+How the Region Aware Implementation Works
+-----------------------------------------
+
+When working with region-aware ReCom chains, it is worth knowing how the spanning tree
+of the dual graph is being split. Weights are randomly assigned to the edges of the graph
+and then the surcharges are applied to the edges in the graph that span different regions
+specified by the ``region_surcharge`` dictionary. So if we have
+``region_surcharge={"muni": 0.2, "water": 0.8}``, then the edges that span different
+municipalities will be upweighted by 0.2 and the edges that span different water districts
+will be upweighted by 0.8. We then draw a minimum spanning tree using Kruskal's algorithm,
+which picks the edges interior to the region first before picking the edges that bridge
+different regions. 
+
+This makes it very likely that each region is largely contained in a connected subtree
+attached to a bridge node. Thus, when we make a cut, the regions attached to the
+bridge node are more likely to be (mostly) preserved in the subtree on either side
+of the cut.
+
+In the implementation of :meth:`~gerrychain.tree.biparition_tree` we further bias this
+choice by deterministically selecting bridge edges first. In the event that multiple
```

Comment:
deterministically cutting?

Reply:
Yeah, that's probably a better word. The function actually selects all of the highest priority edges first and then picks one to be a cut edge, but maybe that wording is confusing for the tutorial docs.